### PR TITLE
Generate rankings only for leagues and adjust the ranking table wording. #125

### PR DESCRIPTION
**Pull request by jonasnoki**

### CODE REVIEW

**Comment by bakoe** [comment]
@jonasnoki Please give the PR a name that represents the actual changes included in the PR better (something like “Generate rankings only for leagues and adjust the ranking table wording”. At the moment, it sounds like the PR introduces the rankings for events which was done in #125 already.

File: app/controllers/leagues_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# This controller extends the league type of events to be capable of showing a ranking entry
+class LeaguesController < EventsController
+  # GET /events/1/ranking
```
Comment (birneamstiel) [inline app/controllers/leagues_controller.rb]: Adjust comment

File: spec/features/event/ranking_event_spec.rb [hunk 2]
```diff
@@ -6,10 +6,6 @@
     @event = FactoryBot.create :rankinglist
```
Comment (birneamstiel) [inline spec/features/event/ranking_event_spec.rb]: Please rename this file to `rankinglist_spec.rb`

File: spec/features/event/ranking_league_spec.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
```
Comment (birneamstiel) [inline spec/features/event/ranking_league_spec.rb]: Please rename this file to `ranking_event_spec.rb` and add negative tests for Tournaments and Rankinglists regarding the generation of Rankings

File: spec/features/event/ranking_league_spec.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    @user = FactoryBot.create :user
+  end
+
+  let(:valid_league_attributes) {
+    FactoryBot.build(:league, owner: @user, max_teams: 20, game_mode: League.game_modes[:round_robin]).attributes
+  }
+
+  it 'should render without an error' do
+    event = League.create! valid_league_attributes
+    visit leagues_ranking_path(event)
```
Comment (birneamstiel) [inline spec/features/event/ranking_league_spec.rb]: Maybe add further assertions and not just visit the page and hope that an error occurs when there is no route(?)

File: spec/routing/events_routing_spec.rb [hunk 5]
```diff
@@ -43,10 +43,6 @@
       expect(get: "/events/1/schedule").to route_to("events#schedule", id: "1")
     end
 
-    it 'routes to #ranking' do
```
Comment (birneamstiel) [inline spec/routing/events_routing_spec.rb:46]: Add negative test:
```ruby
it "doesn't route to #ranking" do
  expect(get: "/events/1/ranking").to_not route_to("events#ranking", id: "1")
end
```

File: spec/views/events/show.html.erb_spec.rb [hunk 6]
```diff
@@ -73,11 +73,6 @@
       render
       expect(rendered).to_not have_selector(:link_or_button, t('helpers.links.destroy'))
     end
-
-    it 'has a ranking button' do
```
Comment (birneamstiel) [inline spec/views/events/show.html.erb_spec.rb:77]: Add negative test:
```ruby
it "doesn't have a ranking button" do		
  render		
  expect(rendered).to_not have_content(t('events.show.to_ranking'))		
end
```

**Review by birneamstiel — CHANGES_REQUESTED** [review]
minor changes :-)

**Review by bakoe — APPROVED** [review]
LGTM code-wise! Have tested together with @birneamstiel whether it works in Browser … it does. 🙂

**Review by birneamstiel — APPROVED** [review]

**Pull request merged**